src/components/app-button-menu/AppButtonMenu.tsx

```diff
@@ -61,12 +60,15 @@ const AppButtonMenu = <T extends Pick<CategoryNameInterface, '_id'>>({
     setSelectedItems([])
   }
 
-  const { loading, response } = useAxios<T[]>({
-    service,
-    defaultResponse: defaultResponses.array
+  const { data: response, isLoading } = useQuery({
+    queryKey: [title],
```

Comment:
Please, add some unique value to the `queryKey`, probably, if `title` will be repeated inside our project as a key, you will have a false positives cache matches
```suggestion
    queryKey: ['app-button-menu', title],
```